src/app/components/Image/index.tsx

```diff
@@ -80,19 +81,20 @@ const Image = ({
   const hasFallback = srcSet && fallbackSrcSet;
   const ImageWrapper = hasFallback ? 'picture' : Fragment;
   const ampImgLayout = hasDimensions ? 'responsive' : 'fill';
-
   const getImgSrcSet = () => {
-    if (!hasFallback) return srcSet;
-    if (pageType !== FRONT_PAGE) return fallbackSrcSet;
-    return undefined;
+    // I don't know if we need this????
+    // if (!hasFallback) return srcSet;
+    // if (pageType !== FRONT_PAGE && pageType !== HOME_PAGE)
+    //   return fallbackSrcSet;
+    // return undefined;
```

Comment:
Shall we just leave this in to be on the safe side?

Reply:
Don't we want to remove this as we want to return a webp srcset by default now? https://github.com/bbc/simorgh/blob/274e3926ec808579c55caeb191e18a6298c87eab/src/app/components/Image/README.md?plain=1#L25

Reply:
If we only know if the browser doesn't support webp by checking the header using the service worker, does that mean the fallback logic needs to be in the service worker? Or I need to check headers in this file too?

This bit:

'
    const supportsWebp =
      req.headers.has('accept') && req.headers.get('accept').includes('webp');'

Reply:
On homepages and frontpages we still use the picture element as per this PR. The picture element will be ignored on old browsers that don't support webp and a jpg request will be sent from the fallback `img` tag. This consequently will mean the correct srcset should be used on old browsers and the fallback `srcset` is included as the `srcset` attribute on the fallback `img` tag.

Now for other pages we are moving to just using the `img` tag with the webp `srcset` as I understand it, the browser will choose an image from the webp `srcset` the service worker will intercept the request for the chosen image and remove the webp extension where necessary.

So in this respect the service work does not need to be aware on `srcsets` unless I've missed something?

ref
> `srcset` defines the set of images we will allow the browser to choose between, and what size each image is. Each set of image information is separated from the previous one by a comma

https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#resolution_switching_different_sizes

Reply:
A discussion with @eagerterrier led to thinking we should leave the picture tag until the problematic iOS version isn't used anymore. This would be 6-12 months. So I haven't removed it in this work.


'The picture element will be ignored on old browsers that don't support webp and a jpg request will be sent from the fallback img tag.'
Does this happen automatically? I don't need to add code to detect the browser not supporting webp?

Reply:
_Note for reviewers had a DM with @LilyL0u and further changes are needed to only render the picture element on home and front pages, this comment assumes this change is being made_

I think we do need to retain this logic as the `img` tag should be the fallback of last resort for old browsers; with this in mind the `img` tag should always use the fallback `srcset` for home and frontpages that use the picture element. Older browsers that don't support the picture element are also unlikely to support service workers and webp so we are better providing a fallback.

On other page types the `img` tag can be used with the primary `srcset` with the service worker intercepting the image request and stripping the `webp` extension as appropriate.